docs/dls/contributing-to-storybook.md

```diff
@@ -4,8 +4,38 @@ title: Contributing to Storybook
 description: Documentation on How to Contribute to Storybook
 ---
 
-You find our storybook at
-[http://backstage.io/storybook](http://backstage.io/storybook)
+Storybook is an open source tool to provide users a way to explore the Backstage
```

Comment:
Feeling that this implies that storybook is built for for this purpose specifically, which isn't quite the case 😅

Reply:
Yep; great point. Will reword.